spec/system/admin/product_import_spec.rb

```diff
@@ -715,7 +715,7 @@ def save_data
 
   def expect_progress_percentages(*percentages)
     percentages.each do |percentage|
-      expect(page).to have_selector ".progress-interface", text: percentage
+      page.has_selector? ".progress-interface", text: percentage # Waits for progress bar
```

Comment:
I think in this case `expect` will also wait, so this change isn't needed.

Reply:
I could not get it to pass locally without this change, I get:

```
  1) Product Import handling a large file (120 data rows) when importing to product list validates and saves all batches
     Failure/Error: expect(page).to have_selector ".progress-interface", text: percentage
       expected to find visible css ".progress-interface" with text "33%" but there were no matches. Also found "", "Save imported products (0%)", which matched the selector but not all filters. 
```